resources/views/components/transaction/section-detail/transaction-type.blade.php

```diff
@@ -0,0 +1,86 @@
+@props([
+    'transaction',
+    'width' => null,
+])
+
+@php
+    $items = [
+        trans('pages.transaction.header.category') => [
+            'component' => '<x-general.encapsulated.transaction-type-badge
+                :transaction="$transaction"
+                class="inline-block"
+            />',
```

Comment:
would this not be nicer to have inside the actual `page-section` component instead of passing the components around? 🤔 or did you run into limitations

Reply:
I'll refactor it since this is messy and thinking about it, unnecessary. I tried to make it "versatile", but in actual fact I can just rely on slots